css/types/timing-function.json

```diff
@@ -153,6 +153,57 @@
               "deprecated": false
             }
           }
+        },
+        "jump": {
```

Comment:
This may be a bit of a continuation of the earlier issue, but again, if I understand correctly, this is a subfeature of `steps` above, rather than a peer of it?

(We'd probably need to adjust the depth argument on the MDN page so that the subsubfeature actually shows.)

Reply:
jump-* is a subfeature of step, but if we make it a subfeature, i was afraid it would never appear in the table. Do subfeatures also show? or would jump, as a sub feature of step, only show on a call to css.properties.animation-timing-function.step? (right now we call css.properties.animation-timing-function)

Reply:
I nested it, and added the ', 2' to the BCD table call in the <timing-function> page.